go/cmd/vttestserver/main.go

```diff
@@ -206,6 +206,7 @@ func (t *topoFlags) buildTopology() (*vttestpb.VTTestTopology, error) {
 // Annoying, but in unit tests, parseFlags gets called multiple times per process
 // (anytime startCluster is called), so we need to guard against the second test
 // to run failing with:
+//
```

Comment:
nit: looks like unrelated line change.

Reply:
> ... or maybe my local tooling is out of date?

actually the opposite; i noticed this after updating Go versions and `golangci-lint`. I actually added this newline in another PR, because golint yelled at me for it (so if you rebase/pull, this diff should actually go away)